app/src/main/res/values-it/strings.xml

```diff
@@ -1,4 +1,56 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-<!--    TODO  add translation for strings file-->
+    <string name="app_name">meteoapp</string>
+    <string name="home_icon">Home</string>
+    <string name="today_icon">Oggi</string>
+    <string name="tomorrow_icon">Domani</string>
+    <string name="search_icon">Search</string>
+    <string name="home_fragment">HOME!</string>
+    <string name="city">Palermo,Sicilia</string>
+    <string name="today">Oggi,</string>
+    <string name="today_date">Domenica 10 settembre</string>
+    <string name="hour_now">11:00</string>
+    <string name="grade_now">31°</string>
+    <string name="humidity_now">0%</string>
+    <string name="perceived">Percepita</string>
+    <string name="UV">Indice UV</string>
+    <string name="humidity">Umidità</string>
+    <string name="wind">Vento</string>
+    <string name="cover">Copertura</string>
+    <string name="rain">Pioggia</string>
+    <string name="grade_perceived">45°</string>
+    <string name="card_UV">5/10</string>
+    <string name="humidity_percentage">60%</string>
+    <string name="wind_speed">SSE 7Km/h</string>
+    <string name="cover_percentage">24%</string>
+    <string name="rain_cm">0cm</string>
+    <string name="hour_for_every_hours">12:00</string>
+    <string name="grade_for_every_hours">29°</string>
+    <string name="humidity_for_every_hours">0%</string>
+    <string name="sun">sole</string>
+    <string name="drop">goccia</string>
+    <string name="hour0">00:00</string>
+    <string name="hour1">01:00</string>
+    <string name="hour2">02:00</string>
+    <string name="hour3">03:00</string>
+    <string name="hour4">04:00</string>
+    <string name="hour5">05:00</string>
+    <string name="hour6">06:00</string>
+    <string name="hour7">07:00</string>
+    <string name="hour8">08:00</string>
+    <string name="hour9">09:00</string>
+    <string name="hour10">10:00</string>
+    <string name="hour11">11:00</string>
+    <string name="hour12">12:00</string>
+    <string name="hour1pm">13:00</string>
+    <string name="hour2pm">14:00</string>
+    <string name="hour3pm">15:00</string>
+    <string name="hour4pm">16:00</string>
+    <string name="hour5pm">17:00</string>
+    <string name="hour6pm">18:00</string>
+    <string name="hour7pm">19:00</string>
+    <string name="hour8pm">20:00</string>
+    <string name="hour9pm">21:00</string>
+    <string name="hour10pm">22:00</string>
+    <string name="hour11pm">23:00</string>
 </resources>
```

Comment:
I left a comment into the other PR about hours and in general for data that belong to the domain model. Why we have those changes into 2 different PR???